content/using/operations/using-your-ship.md

```diff
@@ -570,6 +570,33 @@ Syntax `;unset notify`
 Do not notify when your ship name is mentioned.
 
 
+## S3 {#s3}
+
+Amazon S3, which stands for Amazon Simple Storage Service, is a technology that was introduced by Amazon Web Services for cloud storage that has since been extended more broadly into a [standard programming interface](https://en.wikipedia.org/wiki/Amazon_S3#S3_API_and_competing_services).
+
+Our S3 support lets you store credentials for your S3 object storage buckets on your Urbit ship; once you have done so, you will get additional functionality for uploading your own media within Chat and Groups (for avatars).
+
+### Bucket Setup
+
+If you’re using AWS, it will have to support signature v2, not v4. We've found DigitalOcean to work well also. The bucket has to be publicly readable; allow CORS from `*` origins, and allow `*` headers. Your provider should have details on setting access permissions and CORS.
+
+### Add your credentials: Landscape
+
+Navigate to your `https://<your-ship-url>/~profile/settings`, or click your sigil in the upper-right corner of the window. Fill out the **S3 Credentials** and **S3 Buckets** sections and you're all set!
+
+### Add your credentials: Dojo
```

Comment:
I think it should be clarified that you should use *either* landscape or dojo here; these are alternatives, and you don't have to do both.